GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -986,14 +1008,117 @@ public void ForceRefreshRevisions()
                     }
                 });
                 ResetNavigationHistory();
+
+                return;
             }
             catch
             {
                 SetPage(new ErrorControl());
                 throw;
             }
 
-            return;
+            string BuildPathFilter(string? path)
```

Comment:
❗  We really need to test it.

Reply:
Can we take this later?
There are several PRs that depends on this PR.

The quoting handling is making some guessing, but as the handling ultimately uses Git to ignore bad paths, it should not be critical.
The quoting and --follow handling can be separated.

...and I have raised my opinions about the usefulness about parsing internal methods like this before...

Reply:
> Can we take this later?

Sure



> ..and I have raised my opinions about the usefulness about parsing internal methods like this before...

If we don't have tests, there's a high change we'll either keep churning here trying to fix unintended regressions, or regress while updating the functionality.